woodwork/tests/schema/test_table_schema.py

```diff
@@ -575,14 +575,14 @@ def test_set_logical_types_empty(sample_column_names, sample_inferred_logical_ty
         use_standard_tags=True,
     )
 
-    # An empty set should reset the tags
+    # An empty set should preserve custom tags
```

Comment:
Is this the behavior we want? I was thinking that if a user specifies a new set of tags to use any existing tags should get cleared out. The only time we would leave existing tags alone is when `semantic_tags=None` in the call to `set_types`. If users want to add tags to existing tags they would use the `add_semantic_tags` method.

Reply:
If you haven't started updating already, it might not hurt to get a second opinion to clarify how we want this to work before you spend time updating things you have already done.

Reply:
+1 for `semantic_tags=set()` meaning remove all tags and `semantic_tags=None` mean preserve custom tags

Reply:
updated